bakery/src/pipeline.js

```diff
@@ -116,7 +119,10 @@ module.exports.handler = argv => {
       reportToOutputProducer(Status.PROCESSING),
       taskFetchBook(),
       taskAssembleBook(),
+      taskAssembleBookMetadata(),
       taskBakeBook(),
+      taskDisassembleBook(),
+      taskJsonifyBook(),
       taskMathifyBook(),
```

Comment:
The fact that our content flow branches here has me thinking how this should be reflected in our pipelines. @rnathuji and cc: @philschatz do we think whenever someone builds a pdf of a book, we will want to do something with these JSON artifacts, or is work we want to do triggered by some other event?

Reply:
@tomjw64 - I'd love to discuss this more with you and @philschatz as it's 1) Something I'm unclear on myself as far as the intended interdependencies; 2) I'm still new to Concourse and don't know the best practices well, but this branching makes me wonder if we should be structuring things differently if they're fairly decoupled? For example, right now it feels wrong that if disassemble breaks mathify and beyond never happen.

Reply:
@tomjw64 , @philschatz - Based on some discussions I had with @c0ldcalamari , I'm wondering if for now it doesn't make more sense to just merge the task definitions and leave the pipeline changes out of this PR since some of this may well be TBD and not become clear until we have more clarity from some of the other outstanding tasks (e.g. perhaps there will be a separate pipeline definition altogether for the JSON artifact generation and push of the various files to S3). Let me know if that sounds reasonable to you guys, and I'll update the PR accordingly. Conversely, if the right approach is clear at this stage, we can discuss to be sure we're all on the same page.

Reply:
I think you're right. We should leave the pipeline changes for another day. Approved.